opendevin/runtime/client/runtime.py

```diff
@@ -49,6 +50,7 @@ def __init__(
         plugins: list[PluginRequirement] | None = None,
         container_image: str | None = None,
     ):
+        self.config = copy.deepcopy(config)
```

Comment:
I think this is actually done it is parent constructor `super().__init__()` if I'm not remembering wrong?

Reply:
Oh nvm i think i got it?

> EventStreamRuntime config is output for each test, but Runtime config: only for the first test, thus retaining original values and the 2nd test is still run internally with opendevin instead of root and failing.